### ISSUE TITLE
[BUG] <title>Unsaved files/tabs discarded without warning when switching to Admin Mode

### ISSUE THREAD

**Issue by bcook75** [notepad-plus-plus/notepad-plus-plus#15323]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Issue

I have accumulated 6 or 8 tabs of things that had never been saved in Notepad. One of my most valued features is that all of these tabs are restored every time I restart Notepad++. Today, I had the HOSTS file open. When I tried to save a change, I was told I had to enable admin mode. I was NOT warned that all unsaved tabs would be discarded and all other tabs would be closed without saving. This was quite shocking. It should be handled better.

### Steps To Reproduce

1.  Open a few tabs in Notepad++ in normal more, some of actual files-on-disk, some just to hold unsaved info
2.  Open a protected file, like c:\windows\system32\drivers\etc\hosts
3. Try to save the hosts file
4. Agree to restart Notepad++ in admin mode.

### Current Behavior

Complete loss of some long-accumulated information.

### Expected Behavior

At the very least, a warning about what is going to happen to ALL of the tabs that are currently open, ESPECIALLY the ones that have never been saved as file. Reopening recently closed tabs does NOT appear to recover those unsaved tabs.

### Debug Information

```shell
Notepad++ v8.6.8   (64-bit)
Build time : Jun  4 2024 - 00:30:00
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
Periodic Backup : ON
OS Name : Windows 10 Enterprise (64-bit)
OS Version : 22H2
OS Build : 19045.4412
Current ANSI codepage : 1252
Plugins : 
    mimeTools (3.1)
    NppConverter (4.6)
    NppExport (0.4)
    XMLTools (3.1.1.13)
```

**Comment by xomx** [comment]
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/14990#issuecomment-2090354660
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/14990#issuecomment-2097762215
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/14990#issuecomment-2098512536
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/14990#issuecomment-2115152716

What is your N++ settings for (screenshots welcome):
- Preferences > Backup > Session snapshot and periodic backup
- Preferences > Multi-instance & Date > Multi-instance settings

**Comment by bcook75** [comment]
I have both those settings - snapshot on, multi-instance off.

After I reported the problem I tried to duplicate the problem but it didn't happen again. NP++ had JUST been updated in that session, and had not been open long (minutes, not hours or days) when it happened. 

I happened to be presenting via Teams at the time, and at least 2 other users of NP++ saw it and were just as shocked as me. Been using NP++ for years and never lost a speck of data, so my trust level was pretty much absolute.

Just a glitch, it seems, except for the other feedback from other users.

**Comment by donho** [comment]
@bcook75 
I cannot reproduce the issue.
Have you still encountered such problem?

@xomx 
Can you reproduce it as well?

**Comment by xomx** [comment]
@donho 
No, couldn't reproduce (tried earlier with the user reported v8.6.8).

> 4\. Agree to restart Notepad++ in admin mode.

> NP++ had JUST been updated in that session,

I think (not really tested) that it could be connected somehow to that very specific user situation (a N++ update and restart as admin together).
